zerver/openapi/zulip.yaml

```diff
@@ -1855,6 +1855,9 @@ paths:
                                     has to be muted, the second element is the topic name to be muted
                                     and the third element is an integer UNIX timestamp representing
                                     when the topic was muted.
+
+                                    **Changes**: New in Zulip 6.0 (feature level 131). This is now
```

Comment:
Hmm, `muted_topics` is not new in Zulip 6.0, right? It's just getting a behavior change (that it's no longer sent when `user_topic` is specified as an event type).

And, more concretely, does "specified as an event type" mean "present in `fetch_event_types`"? Or does it mean "present in `fetch_event_types` or `event_types`", or something else?

Reply:
This should state, " `muted_topics` data will not be returned if `user_topic` is included alongside it, in either `fetch_event_types` or `event_types` ". (As events in `event_types` are used to fetch initial state data, if `fetch_event_types` is empty.)